client/components/timezone/index.jsx

```diff
@@ -60,16 +61,24 @@ class Timezone extends Component {
 	}
 
 	render() {
-		const { selectedZone } = this.props;
+		const { className, selectedZone } = this.props;
 		return (
-			<select onChange={ this.onSelect } value={ selectedZone || '' }>
+			// Changing this to onBlur will not work, so we ignore this
```

Comment:
I think that regardless of the reasons behind not being able to migrate `onChange` to `onBlur`, we shouldn't encourage this and should be looking into fixing it in the future. Personally I'd pay the price and accept single failing tests (because of eslines) in favor of not adding an `eslint-disable` rule when it indicates a real a11y flaw.

In case you decide to go with the rule ignore, could you elaborate in the inline comment and explain why it won't work? Possibly add a `TODO` too? It could make a good case to revisit in the future.